docs/settings.rst

```diff
@@ -774,6 +774,16 @@ be filtered out.
 
 For example: ``[(logging.WARN, 'TAG_SAVE_AS is set to False')]``
 
+It's possible to filter out messages by a template. Check out source code to obtain a template.
+
+For example: ``[(logging.WARN, 'Empty alt attribute for image %s in %s')]``
+
+**Warning:** Silence messages by templates is dangerous feature. There are odds
+to unintantional filter out messages of different types with the same template
+(includes future messages from newer Pelican versions). Proceed with caution.
+
+Note: This option do nothing when ``--debug`` passed.
+
```

Comment:
Some grammar suggestions:

**Warning:** Silencing messages by templates is a dangerous feature. It is possible to unintentionally filter out multiple message types with the same template
(including messages from future Pelican versions). Proceed with caution.